src/telemetry/telemetry.c

```diff
@@ -226,10 +254,15 @@ add_job_counts(JsonbParseState *state)
 	BgwJobTypeCount counts = bgw_job_type_counts();
 
 	ts_jsonb_add_int32(state, REQ_NUM_POLICY_CAGG, counts.policy_cagg);
+	ts_jsonb_add_int32(state, REQ_NUM_POLICY_CAGG_FIXED, counts.policy_cagg_fixed);
```

Comment:
Does it make sense to define macros for these names? They're used in one place, another layer of indirection just makes it harder to read.

Reply:
I think defining the macro follows the same pattern as for the rest of the parameters, so makes sense to do it for that reason.